src/libostree/ostree-bootloader-grub2.c

```diff
@@ -30,6 +30,7 @@
 #define BOOTUPD_CONFIG "boot/bootupd-state.json"
 // Horrible hack, to avoid including a JSON parser we just grep for this
 #define BOOTUPD_CONFIG_STATIC_JSON_FRAGMENT "\"static-configs\""
+#define BOOTUPD_CONFIG_STATIC_JSON_FRAGMENT_NULL "\"static-configs\":null"
```

Comment:
Ouch :cry: 

I dunno, maybe overdue for us to link in a json library, I had just hoped to do that with rust (that's in https://github.com/ostreedev/ostree/pull/3054 but nontrivial)

Reply:
Yeah, I agree it's really not great :/